x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/import_jobs_flyout.tsx

```diff
@@ -0,0 +1,470 @@
+                  <EuiSpacer size="m" />
+
+                  {jobIds.map((jobId, i) => (
+                    <div key={i}>
+                      <EuiPanel hasBorder={true}>
+                        <EuiFlexGroup>
+                          <EuiFlexItem>
+                            <EuiFormRow
+                              error={jobId.invalidMessage}
+                              isInvalid={jobId.invalidMessage.length > 0}
+                            >
+                              <EuiFieldText
+                                prepend={i18n.translate(
+                                  'xpack.ml.importExport.importFlyout.jobId',
+                                  {
+                                    defaultMessage: 'Job ID',
+                                  }
+                                )}
+                                disabled={importing}
+                                compressed={true}
+                                value={jobId.id}
+                                onChange={(e) => renameJob(e.target.value, i)}
+                                isInvalid={jobId.valid === false}
+                              />
+                            </EuiFormRow>
+                          </EuiFlexItem>
+                          <EuiFlexItem grow={false}>
+                            <DeleteJobButton index={i} />
+                          </EuiFlexItem>
+                        </EuiFlexGroup>
+                      </EuiPanel>
+                      <EuiSpacer size="m" />
+                    </div>
+                  ))}
+                </>
+              )}
+            </>
+          </EuiFlyoutBody>
+          <EuiFlyoutFooter>
+            <EuiFlexGroup justifyContent="spaceBetween">
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty iconType="cross" onClick={() => setShowFlyout(false)} flush="left">
+                  <FormattedMessage
+                    id="xpack.ml.importExport.importFlyout.closeButton"
+                    defaultMessage="Close"
+                  />
+                </EuiButtonEmpty>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton disabled={importDisabled} onClick={onImport} fill>
+                  <FormattedMessage
+                    id="xpack.ml.importExport.importFlyout.closeButton.importButton"
+                    defaultMessage="Import"
+                  />
+                </EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlyoutFooter>
+        </EuiFlyout>
+      )}
+    </>
+  );
+};
+
+const FlyoutButton: FC<{ isDisabled: boolean; onClick(): void }> = ({ isDisabled, onClick }) => {
+  return (
+    <EuiButtonEmpty
+      iconType="importAction"
+      onClick={onClick}
+      isDisabled={isDisabled}
+      data-test-subj="mlJobWizardButtonPreviewJobJson"
+    >
+      <FormattedMessage id="xpack.ml.importExport.importButton" defaultMessage="Import jobs" />
```

Comment:
After the import completes is it possible to auto refresh the list, and / or show a success toast? Otherwise there is no indication that the import has been successful until the user hits the Refresh button.

Reply:
https://github.com/elastic/kibana/pull/101037/commits/5c96c0d9ebbf716d5810d29023c7d17628a01f86 adds a success toast
It's not an easy change to auto-refresh the jobs lists as they exist nested inside other components.
If auto-refresh is needed it would be best to add it in a follow up as it'd need restructuring our main management page.